package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "pattern-replace-loader",
-  "version": "1.0.4",
-  "description": "pattern replace loader for webpack 2",
+  "name": "improved-pattern-replace-loader",
```

Comment:
[q] Why are we changing the name of the package? I guess you did have a fork of it.

Reply:
Yes, I have enabled that maintainers can edit the PR. The package.json is only different because I created a fork of it.